editor.planx.uk/src/pages/Team/helpers/sortAndFilterOptions.ts

```diff
@@ -44,3 +50,14 @@ export const filterOptions: FilterOptions<FlowSummary>[] = [
     validationFn: checkFlowServiceType,
   },
 ];
+
+const templateFilterOption: FilterOptions<FlowSummary> =  {
+  displayName: "Template",
+  optionKey: "templatedFrom",
+  optionValue: ["template"],
+  validationFn: checkFlowTemplatedFrom,
+};
```

Comment:
I'm not sure just "Template" really works here without context, something we can think about!

Reply:
I do wonder if "Template" can be an option under "Service type" rather than it's own third category with same-name option?

Reply:
This was also my first thought, but these options are always exclusive.

https://github.com/user-attachments/assets/995b4fa2-a661-48e6-9d81-95cb5489ca7b

Not ideal and not a simple change after a little while looking at this. A lot of the filter code is pretty complex and I'd love to actually revisit some of this, especially compared to how easy and powerful it was to use the MUI datagrid API. I think there's also a much better general solution here using something like tanstack query (filtering server side, not client side).